infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationDiscoveryService.java

```diff
@@ -343,8 +343,13 @@ private void bootstrapLogReplicationService() {
 
         logReplicationServerHandler = new LogReplicationServer(serverContext, logReplicationConfig,
             logReplicationMetadataManager, localCorfuEndpoint, topologyDescriptor.getTopologyConfigId(), localNodeId);
-        logReplicationServerHandler.setActive(localClusterDescriptor.getRole().equals(ClusterRole.ACTIVE));
-        logReplicationServerHandler.setStandby(localClusterDescriptor.getRole().equals(ClusterRole.STANDBY));
+        if (localClusterDescriptor.getRole().equals(ClusterRole.ACTIVE)) {
+            logReplicationServerHandler.setActive(true);
+            addDefaultReplicationStatus();
```

Comment:
bootstrapLogReplicationService gets invoked on every restart but the state should not be initialized every time.  We will need a check to add this status only for clusters not present in the table.